pkg/secrets/fetch_secret.go

```diff
@@ -148,6 +148,11 @@ func fetchSecret(secretsHandle []string) (map[string]string, error) {
 		if v.ErrorMsg != "" {
 			return nil, fmt.Errorf("an error occurred while decrypting '%s': %s", sec, v.ErrorMsg)
 		}
+
+		if removeTrailingLinebreak {
+			v.Value = strings.TrimRight(v.Value, "\n")
```

Comment:
Is there a reason for not trimming `\r` in this version ? (unlike in https://github.com/DataDog/datadog-agent/pull/16095)

Reply:
not at all, good catch !